boards/m5stack/m5stack_cores3/Kconfig

```diff
@@ -3,5 +3,6 @@
 
 config HEAP_MEM_POOL_ADD_SIZE_BOARD
 	int
-	default 4096 if BOARD_M5STACK_CORES3_ESP32S3_PROCPU
+	default 4096 if BOARD_M5STACK_CORES3_ESP32S3_PROCPU || \
+			BOARD_M5STACK_CORES3_ESP32S3_PROCPU_SE
 	default 256 if BOARD_M5STACK_CORES3_ESP32S3_APPCPU
```

Comment:
@sylvioalves did you mean to actually approve the PR then?

Reply:
Yes, I did. It can be implemented as is for `appcpu`. As `boards.yml` only implements the `se` version for `procpu` variant, no need for the `appcpu`.